integration-tests/oauth2/pom.xml

```diff
@@ -121,7 +121,7 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-failsafe-plugin</artifactId>
-                    <version>2.22.2</version>
+                    <version>${maven.failsafe.plugin.version}</version>
```

Comment:
This version is already defined in one of the parent poms, you _should_ be able to remove this line:
https://github.com/okta/okta-java-parent/blob/master/parent/pom.xml#L210-L234
```suggestion
```